src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -185,26 +185,16 @@ const addInvoiceFiatDenomiation = async ({
 
   const expiresAt = invoiceExpirationForCurrency(WalletCurrency.Usd, new Date())
 
-  // TODO: ensure we don't get a stalled price // fail otherwise
-  const price = await getCurrentPrice()
-  if (price instanceof Error) return price
-
-  const dCConverter = DisplayCurrencyConverter(price)
-
   const dealer = DealerPriceService()
 
-  const amountConverter = AmountConverter({ dealerFns: dealer, dCConverter })
```

Comment:
Can we delete `AmountConverter` and associated unit tests now that it seems we no longer use it?

Reply:
yes I think it's safe to remove them. Maybe I would still keep the unit test to test the mock of the dealer

Reply:
Ok we should leave for now then